ui-v2/tests/acceptance/dc/list-blocking.feature

```diff
@@ -5,11 +5,6 @@ Feature: dc / list-blocking
   I want to see changes if I change consul externally
   Background:
     Given 1 datacenter model with the value "dc-1"
-    And settings from yaml
```

Comment:
This is the removal of the previous setting and the blocking test which should now pass with the default, I think.

Reply:
Yup exactly, I think there 'might' be a few other places in the tests where I specifically set it to `on` before running the tests, that will now essentially be a no-op now. I don't mind those still being in there though, I just removed this one to make sure it definitely works when you haven't set anything to `on`.